### ISSUE TITLE
Unable to boot in safe mode

### ISSUE THREAD

**Issue by 82ghost82** [acidanthera/bugtracker#1967]
Hi,
I tried to boot into safe mode monterey 12.2.1, OC DBG-079-2022-03-06 without success, prohibited symbol.
Running monterey in qemu with ovmf compiled from audk.
Opencanopy loads, I select monterey main volume, press shift+enter --> prohibited symbol.

In the log it seems it fails with:
29:552 10:052 OC: Boot failed - Aborted
29:569 00:016 OCB: StartImage failed - Aborted

What I tried:
1. EnableSafeModeSlide=true, ProvideCustomSlide=true --> prohibited symbol, then it goes back to the picker
2. EnableSafeModeSlide=true, ProvideCustomSlide=true, AllowRelocationBlock=true, AvoidRuntimeDefrag=true (even if from what I understand it may not be compatible with mac os >=11) --> vm pauses with apple logo on the screen, can't resume, only reset
3. Detached the secondary windows hd attached to the same sata controller, so that the only disk attached is that of mac os (n.b.: I need a third party kext to be injected for the sata controller)

This log refers to (1):
[opencore-2022-03-06-125721.txt](https://github.com/acidanthera/bugtracker/files/8192942/opencore-2022-03-06-125721.txt)

Log ends with the nvram reset, the failure happens just before.

Really hope it's not a config issue to not waste your time.

**Comment by vit9696** [comment]
Yes, on some firmware it is indeed not possible to boot in safe mode nowadays starting with macOS 11. It is a wontfix, sorry.

**Comment by perez987** [comment]
@82ghost82 
Adding -x in config.plist boot-args I can boot in safe-mode correctly.
But pressing keys on the picker (Shift Enter...) safe-mode boot ends in black screen.
